.github/workflows/legacy.yml

```diff
@@ -0,0 +1,106 @@
+name: Legacy Torch CI
```

Comment:
This is a copy of the `.github/workflows/ci.yml` workflow, but renamed. That workflow is being repurposed in https://github.com/mojotech/torch/pull/286